packages/server/graphql/public/mutations/removeMultipleOrgUsers.ts

```diff
@@ -0,0 +1,104 @@
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {BATCH_ORG_USER_REMOVAL_LIMIT} from '../../../postgres/constants'
+import {getUserId, isSuperUser, isUserBillingLeader} from '../../../utils/authorization'
+import publish from '../../../utils/publish'
+import standardError from '../../../utils/standardError'
+import isValid from '../../isValid'
+import removeFromOrg from '../../mutations/helpers/removeFromOrg'
+import {MutationResolvers} from '../resolverTypes'
+const removeMultipleOrgUsers: MutationResolvers['removeMultipleOrgUsers'] = async (
+  _source,
+  {userIds, orgId},
+  {authToken, dataLoader, socketId: mutatorId}
+) => {
+  const operationId = dataLoader.share()
+  const subOptions = {operationId, mutatorId}
+
+  if (userIds.length === 0) {
+    return standardError(new Error('Empty userIds array is provided'), {
+      userId: getUserId(authToken)
+    })
+  }
+  if (userIds.length > BATCH_ORG_USER_REMOVAL_LIMIT) {
+    return standardError(
+      new Error(`Cannot remove more than ${BATCH_ORG_USER_REMOVAL_LIMIT} org users at once`),
+      {userId: getUserId(authToken)}
+    )
+  }
+
+  // Validation
+  const viewerId = getUserId(authToken)
+  if (userIds.includes(viewerId)) {
+    return standardError(new Error('Cannot remove yourself'), {userId: viewerId})
+  }
+  if (!(await isUserBillingLeader(viewerId, orgId, dataLoader)) && !isSuperUser(authToken)) {
+    return standardError(new Error('Must be the organization leader'), {userId: viewerId})
+  }
+
+  const organization = await dataLoader.get('organizations').load(orgId)
+  if (!organization) {
+    return standardError(new Error('Organization not found'), {userId: viewerId})
+  }
+
+  const orgUserPromises = userIds.map((userId) =>
+    dataLoader.get('organizationUsersByUserIdOrgId').load({userId, orgId})
+  )
+  const orgUsers = await Promise.all(orgUserPromises)
```

Comment:
+1 this could be 1 line, fewer variables to keep track of. We could also use the `loadMany` function of dataloader, but shouldn't make a performance difference.